.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module'
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:ember-suave/recommended'
```

Comment:
One advantage to me is that you only need to update the top of the file and change from destructuring to imports...